src/pages/my_page/user_profile/MyProjectRegistrations.tsx

```diff
@@ -26,13 +26,13 @@ export const MyProjectRegistrations = ({
   const user = useSelector((store: RootState) => store.user);
   const cristinIdentifier = getIdentifierFromId(user?.cristinId ?? '');
 
-  const [page, setPage] = useState(0);
+  const [page, setPage] = useState(1);
```

Comment:
Noen grunn til at denne er endret fra 1 til 0? Mui-docs sier at page er 0-indeksert, og basert på litt console.log ser det ut som at page settes til 0 likevel, så det gir bare en ekstra "bom"-render 🤔 
```suggestion
  const [page, setPage] = useState(0);
```

Reply:
Ingen spesiell grunn. Ble prøvd mye forskjellig på slutten av dagen i går 😬

Reply:
Skjønner! Ofte lurt å gå gjennom egen PR med en liten lupe i etterkant, da det er veldig lett å glemme sånne ting :D